python/tests/labels.py

```diff
@@ -202,3 +210,11 @@ def test_arange_incorrect_arguments():
         Labels.arange(0.0, 1.0, 2)
     with pytest.raises(ValueError, match="integer"):
         Labels.arange("dummy", 0, 5, 0.2)
+    with pytest.raises(ValueError, match="argument"):
+        Labels.arange("dummy", random=10)
+    with pytest.raises(EquistoreError, match="label name"):
+        Labels.arange(0, 1, 2)
+    with pytest.raises(ValueError, match="integer"):
+        Labels.arange(name="dummy", start=0.0, stop=1.0, step=2)
+    with pytest.raises(ValueError, match="integer"):
+        Labels.arange("dummy", start=0, stop=0.2)
```

Comment:
Could you update these to use the full error message?

Reply:
Sure, done